### PR TITLE
Add design and logo

### PR DESCRIPTION
**Pull request by bchapuis**

### CODE REVIEW

File: _sass/hydeout/_layout.scss [hunk 1]
```diff
@@ -25,7 +25,7 @@
 body {
   background-attachment: fixed;
   background-color: $sidebar-bg-color;
-  background-image: linear-gradient(to bottom, lighten($sidebar-bg-color, 7%), darken($sidebar-bg-color, 7%));
+  // background-image: linear-gradient(to bottom, lighten($sidebar-bg-color, 7%), darken($sidebar-bg-color, 7%));
```
Comment (Yahia1018) [inline _sass/hydeout/_layout.scss:28]: I found the render with white background on computers nice, but not on mobile/tablet, the black background made a separation between the menu and the rest.

File: _sass/hydeout/_layout.scss [hunk 2]
```diff
@@ -36,8 +36,12 @@ body {
   flex: 0 0 auto;
   padding: $section-spacing;
 
+  .logo {
+    width: 180px;
+    margin-bottom: 20px;
```
Comment (Yahia1018) [inline _sass/hydeout/_layout.scss:41]: I would add a little bit more margin-bottom, like 25px or in rem 1.5rem, to be symetric with the padding above the logo. I found it nicer like that.

File: _sass/hydeout/_layout.scss [hunk 3]
```diff
@@ -106,7 +110,7 @@ body {
 .home #sidebar {
 
   // Center sidebar content
-  text-align: center;
+  // text-align: center;
```
Comment (Yahia1018) [inline _sass/hydeout/_layout.scss:113]: I found it better with "text-align: center" on mobile, and maybe with the logo in the center too ? And so, even without the black background, we could make a difference with menu and the rest of the page.

File: _sass/hydeout/_layout.scss [hunk 4]
```diff
@@ -255,8 +259,7 @@ body {
 }
 
 #sidebar a:hover,
-#sidebar a:focus,
-#sidebar .site-title:hover {
```
Comment (Yahia1018) [inline _sass/hydeout/_layout.scss:259]: There was too much red ?

File: _sass/hydeout/_variables.scss [hunk 5]
```diff
@@ -42,9 +42,9 @@ $code-font-family: Menlo, Monaco, 'Courier New', monospace !default;
 $code-color: #bf616a !default;
 
 // Hyde theming
-$sidebar-bg-color: #202020 !default;
-$sidebar-fg-color: #fff !default;
-$sidebar-sticky: true !default;
+$sidebar-bg-color: #fff !default;
+$sidebar-fg-color: #000 !default;
+$sidebar-sticky: false !default;
```
Comment (Yahia1018) [inline _sass/hydeout/_variables.scss:47]: I like the idea of the sidebar on the top.

File: _sass/hydeout/_layout.scss (hunk 1 above)
Comment (bchapuis) [reply _sass/hydeout/_layout.scss:28]: The heig-vd logo does not render well on a black background. Lets find a solution for this later.

Comment (Yahia1018) [reply _sass/hydeout/_layout.scss:28]: I purposed that in an another comment :

> I found it better with "text-align: center" on mobile, and maybe with the logo in the center too ? And so, even without the black background, we could make a difference with menu and the rest of the page.

**Review by Yahia1018 — CHANGES_REQUESTED** [review]
There are some propositions you didn't answer.

File: _sass/hydeout/_layout.scss (hunk 1 above)
Comment (Yahia1018) [reply _sass/hydeout/_layout.scss:28]: Capture d’écran 2022-04-06 à 15 04 49

And with this background color ? As the repository logo.

File: _sass/hydeout/_layout.scss (hunk 2 above)
Comment (bchapuis) [reply _sass/hydeout/_layout.scss:41]: Please go ahead with the space you like ;)

File: _sass/hydeout/_layout.scss (hunk 4 above)
Comment (bchapuis) [reply _sass/hydeout/_layout.scss:259]: yes

**Review by Yahia1018 — APPROVED** [review]

**Pull request merged**